### DISCUSSION TITLE
No option to add an issue when viewing issues after clicking on the Issues shortcut from the main Home Screen

### DISCUSSION THREAD

**Discussion by rlwakefield** [community/community#75285] [Mobile]
I tapped on the Issues shortcut on the Home Screen and was taken to all of the issues on the repositories I own. I then went to filter on a specific repository and noticed that there wasn't an issue reported yet for what I was experiencing. So I went to find the "plus" button to add a new issue and noticed there wasn't an option to do so.

So in order to add the new issue, I had to go back to the Home Screen, click the plus button, choose the repository I wanted to create the issue under, and then finally create the issue. Alternatively, I could also go to my account Screen, click on repositories, choose the repository I needed to create the issue in, then click on Issues for that repository, and finally create the Issue needing to be created.

It would be nice if the plus button that allows for the creation of an issue to be visible when you are viewing issues across ALL of the repositories that you own.

**Comment by HenSquared** [comment]
Hey @rlwakefield , thanks a lot for the feedback. It would be nice to be able to add Issues from the Issues screeen. 

I've added this feature request to be tracked and discussed internally. If we have updates on this in the future, we will reply again to this thread. 

Thanks :)

**Reply by rlwakefield** [reply]
Thank you. I look forward to any updates you provide over time. :)
